### DISCUSSION TITLE
Flex direction

### DISCUSSION THREAD

**Discussion by Brenerss** [tailwindlabs/tailwindcss#11973] [Help] [answered]
Im using Nextjs 13 with tailwindcss 3.3.3

for some reason "flex flex-col" dont apply flex direction in my code

```
const GroupListSidebar = () => {
  return (
    <aside className="flex flex-col">
      <Button title="Create" />
      <ul className="text-lighter">
        <li>
          <Link href="#">
            <span></span>
            All Groups
          </Link>
        </li>
        <li>
          <Link href="#">
            <span></span>
            My groups
          </Link>
        </li>
        <li>
          <Link href="#">
            <span></span>
          </Link>
        </li>
      </ul>
    </aside>
  )
}

```

or a simple code

```
<div className="flex flex-col">
      <GroupCard />
      <GroupCard />
      <GroupCard />
      <GroupCard />
    </div>

```

**Comment by wongjn** [comment, marked as answer]
Consider checking:
- Tailwind has compiled since you added the class.
- The `content` file globs sufficiently cover the file that the code appears in.
- [String concatenation/interpolation is not used to build the class name](https://tailwindcss.com/docs/content-configuration#dynamic-class-names). 
- The class name would resolve to a Tailwind class that Tailwind would generate a rule for.

**Reply by Brenerss** [reply]
The problem was solved, I needed to put it in 'content'

` './src/features/**/*.{js,ts,jsx,tsx,mdx}' `
